nl_NL/LC_MESSAGES/discordbot.po

```diff
@@ -10,27 +10,27 @@ msgstr ""
 
 # Username was given a Nitro Booster Ticket...
 msgid "BOOST_TICKET_SUCCESS"
-msgstr "%s was given a Nitro Booster Ticket, redeemable for a month of Sudomemo Plus and a special trophy. Hooray!"
+msgstr "%s Kreeg een nitro boost ticket voor een maand van sudomemo! Hoera!"
 
 # Account linking
 msgid "DISCORD_LINKING_GET_STARTED"
-msgstr "Please visit %s to get started!"
+msgstr "Bekijk %s om te beginnen!"
 
 msgid "DISCORD_LINKING_REQUIRED"
-msgstr "You'll need to link your Sudomemo account to do that."
+msgstr "Je moet je sudomemo account koppelen om dit te doen!"
```

Comment:
The first letter of "Sudomemo" should be capitalized, "!" should be changed back to "."